snippets/olmv1-on-cluster-catalog-queries.adoc

```diff
@@ -0,0 +1,7 @@
+// Text snippet included in the following modules:
+//
+// * olmv1-finding-operators-to-install.adoc
+
+:_mod-docs-content-type: SNIPPET
+
+Currently, {olmv1-first} does not support querying on-cluster catalogs managed by catalogd. In OLM v1, you must use the `opm` and `jq` CLI tools to query the catalog registry.
```

Comment:
I am not sure if I got this right. And if I got it right, I am not sure if this is the best phrasing. Please let me know your thoughts, Operator Framework team.